zephyr/wrapper.c

```diff
@@ -476,6 +476,8 @@ static inline const void *smex_placeholder_f(void)
  */
 const void *_smex_placeholder;
 
+static int w_core_enable_mask;
```

Comment:
uncached ? I assume yes with linker ?

Reply:
@lgirdwood Ack, this is uncached. Even double-checked on an actual device.